### ISSUE TITLE
[VT-97] [Patch] vitrc.5 man page patch

### ISSUE THREAD

**Issue by tbabej** [tbabej/vit#97]
_Ankur Sinha on 2014-06-24T04:05:26Z says:_ 

 *  Fix the vitrc man page

**Comment by tbabej** [comment]
Migrated metadata:
```
Created: 2014-06-24T04:05:26Z
Modified: 2014-06-25T19:48:26Z
```

**Comment by tbabej** [comment]
_Scott Kostyshak on 2014-06-25T19:48:26Z says:_ 

 Fixed at d7f928fb. Thanks Ankur.
